src/main/java/com/autotune/analyzer/serviceObjects/CreateExperimentAPIObject.java

```diff
@@ -0,0 +1,206 @@
+    }
+
+    public void setPerformanceProfile(String performanceProfile) {
+        this.performanceProfile = performanceProfile;
+    }
+
+    public String getMode() {
+        return mode;
+    }
+
+    public void setMode(String mode) {
+        this.mode = mode;
+    }
+
+    public String getTargetCluster() {
+        return targetCluster;
+    }
+
+    public void setTargetCluster(String targetCluster) {
+        this.targetCluster = targetCluster;
+    }
+
+    public List<KubernetesObject> getKubernetesObjects() {
+        return kubernetesObjects;
+    }
+
+    public void setKubernetesObjects(List<KubernetesObject> kubernetesObjects) {
+        this.kubernetesObjects = kubernetesObjects;
+    }
+
+    public TrialSettings getTrialSettings() {
+        return trialSettings;
+    }
+
+    public void setTrialSettings(TrialSettings trialSettings) {
+        this.trialSettings = trialSettings;
+    }
+
+    public RecommendationSettings getRecommendationSettings() {
+        return recommendationSettings;
+    }
+
+    public void setRecommendationSettings(RecommendationSettings recommendationSettings) {
+        this.recommendationSettings = recommendationSettings;
+    }
+
+    public SloInfo getSloInfo() {
+        return sloInfo;
+    }
+
+    public void setSloInfo(SloInfo sloInfo) {
+        this.sloInfo = sloInfo;
+    }
+
+    @Override
+    public String toString() {
+        return "CreateExperimentSO{" +
+                "clusterName='" + clusterName + '\'' +
+                ", performanceProfile='" + performanceProfile + '\'' +
+                ", mode='" + mode + '\'' +
+                ", targetCluster='" + targetCluster + '\'' +
+                ", kubernetesObjects=" + kubernetesObjects +
+                ", trialSettings=" + trialSettings +
+                ", recommendationSettings=" + recommendationSettings +
+                '}';
+    }
+}
+class KubernetesObject {
+    private String type;
+    private String name;
+    private String namespace;
+    private List<Container> containers;
+
+    public KubernetesObject(String name, String type, String namespace) {
+        this.name = name;
+        this.type = type;
+        this.namespace = namespace;
+    }
+
+    // getters and setters
+
+    public String getType() {
+        return type;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getNamespace() {
+        return namespace;
+    }
+
+    public List<Container> getContainers() {
+        return containers;
+    }
+
+    public void setContainers(List<Container> containers) {
+        this.containers = containers;
+    }
+
+    @Override
+    public String toString() {
+        return "KubernetesObject{" +
+                "type='" + type + '\'' +
+                ", name='" + name + '\'' +
+                ", namespace='" + namespace + '\'' +
+                ", containers=" + containers +
+                '}';
+    }
+}
+
+class Container {
```

Comment:
This should be called `ContainerAPIObject`, add a javadoc

Reply:
Done